studio/utils.py

```diff
@@ -0,0 +1,31 @@
+import logging
+from typing import List
+
+import structlog
+from django.conf import settings
+
+
+def get_logger(name: str):
+    """
+    Get different loggers depending on the value of DEBUG.
+    When DEBUG = True, then we return the standard logger,
+    otherwise, the structlog.
+    """
+    if settings.DEBUG:
+        return logging.getLogger(name)
+    else:
+        return structlog.getLogger(name)
+
+
+def add_loggers(logging: dict, installed_apps: List[str]) -> dict:
+    """
+    Helper function to add loggers to each installed app
+    """
+    for apps in installed_apps:
+        logging["loggers"][apps] = {
+            "handlers": ["console" if settings.DEBUG else "json"],
+            "level": "DEBUG" if settings.DEBUG else "INFO",
+            "propagate": False,
+        }
+
+    return logging
```

Comment:
The idea here it to set all dependent packages log-level to WARNING, then add log-level DEBUG to all installed apps. This is to reduce the amount of messages received from third party stuff.  What do you think?

Reply:
If there is no better solution, then yes, it seems like a good idea to me

One thing though, maybe it would be more verbose to not make this a function and add this directly to settings file as a for loop?

Reply:
Yeah maybe! But then we must update the configmap in the charts even more. Maybe that is fine. Ill consider it!:)